README.md

```diff
@@ -57,4 +96,45 @@ List of valid categories:
  15. Themes
  16. Other
 
-The official list of downloadable programs is kept on a public ReactOS server and synced every time RAPPS is launched for the first time.
+### URLSite
+Main web site where the program can be found. **__ Is HTTPS mandatory or only advised? If the link is in HTTP, can it try to switch automatically to HTTPS if avaliable? __**
+
+### URLDownload
+Direct link to the installer/program. **__ Is HTTPS mandatory or only advised? If the link is in HTTP, can it try to switch automatically to HTTPS if avaliable? __**
+
+### ScreenshotN
+Screenshot URL link. (N = 1, 2, 3 ...)
+
+Currently, only first Screenshot is used. Ensure the URL is a direct link to the image, not a page containing the image (e.g. Imgur links). It is also preferred to include the screenshot taken on ReactOS, not on Windows.  **__ Does it still support only one screenshot? Have you thought about creating a repository for screenshots? __**
+
+### Icon
+Icon filename in icons folder (with .ico extension) **__ Any specification on size etc? __**
+
+### DEPRECATED
+Previously used parameters, now deprecated, should be removed if found in a existing file
+
+* Size
+
+### EXPERIMENTAL
+
+* Languages
+* LicenseInfo
+
+### Languages
+This field is used to inform the user whether the app is available in their language.
+
+You should place all the language codes app supports separated by | there. 
+Example
+     Languages=0C09|0813|0422
+**__ How are these code formatted? Are the same used in Sections? __**     
```

Comment:
see the ones already doing this....

Reply:
Ok, apparently same language codes. Are users advised to add the Languages parameter or to avoid it because it is experimental?